bambi/priors/scaler.py

```diff
@@ -58,23 +65,66 @@ def scale_response(self):
     def scale_intercept(self, term):
         if term.prior.name != "Normal":
             return
-        mu, sigma = self.get_intercept_stats()
+        # Special case for logit links with bernoulli family
+        if (
+            isinstance(self.model.family, (Bernoulli, Binomial))
+            and self.model.family.link["p"].name == "logit"
+        ):
+            mu = 0
+            sigma = 1.5
+        else:
+            mu, sigma = self.get_intercept_stats()
         term.prior.update(mu=mu, sigma=sigma)
 
     def scale_common(self, term):
         if term.prior.name != "Normal":
             return
 
-        # It can be greater than 1 for categorical variables
         if term.data.ndim == 1:
             mu = 0
-            sigma = self.get_slope_sigma(term.data)
+            if (
+                isinstance(self.model.family, (Bernoulli, Binomial))
+                and self.model.family.link["p"].name == "logit"
+            ):
+                if term.kind == "interaction":
```

Comment:
Can we add a couple of comments saying what we are checking here? 

For example, whether it's a single categorical variable, an interaction between categoricals (which we treat as a single categorical for this purpose), an interaction between numeric(s) and categorical(s), or just a numeric or interaction between numerics (which we treat as a single numeric for this purpose).